cli/src/main/java/org/adamalang/cli/runtime/ArgumentItem.java

```diff
@@ -0,0 +1,32 @@
+package org.adamalang.cli.runtime;
+
+import java.util.Map;
+
+public class ArgumentItem {
+    public String value;
+    public String name;
+    public String shortField;
+    public String documentation;
+    public String defaultArg = "";
+    public boolean optional = false;
```

Comment:
finalize for immutability; this will have consequences in the constructor will need to group for every aspect. However, aiming to finalize everything generally leads to better practices later on.